playbooks/deployments.md

```diff
@@ -94,21 +94,35 @@ We prefer to use our CI pipeline to "roll forward" even for reverts or timely fi
 
 When it's critical to avoid the delay of a full CI run or staging deploy, you can hot-fix/roll-back using Hokusai, as follows:
 
-- Show a list of the project's ECR images tagged with `production-...`.
+- List the project's production ECR images.
+
   ```
-  hokusai registry images --filter-tags production
+  $ hokusai registry images --filter-tags production
+
+  Image Pushed At     | Image Tags
+  ----------------------------------------------------------
+  2022-01-27 12:15:21-05:00 | production, staging--2022-01-27--19-05-54, 9718ddb9334c3e9b2a0a0ffa5d744e1ca91d5cb3, production--2022-01-27--19-43-45
+  2022-01-26 06:16:16-05:00 | staging--2022-01-26--11-55-47, production--2022-01-26--14-17-47, 84fd6dcd9b115482e2b1d2981c31f4c8bc97a015
+  2022-01-25 11:02:42-05:00 | fcf109fa7db52c538755a4eac1b103ecf83dddce, staging--2022-01-25--16-57-31, production--2022-01-25--17-54-15
+  ...
+
+  81 more images available
   ```
 
-- From the list, identify the last good working image - the one you want to revert to. Note it's tag.
+  The one being used in Production right now is the one that has the canonical `production` tag. In the example output above, it's the first one in the list. Presumably, this is the image that is problematic. The other images that have `production-<timestamp>` tags (but not the canonical `production` tag) were used in Production in the past but are no longer. Presumably one of them is the known last good working image.
+
+- Identify the tag of the known last good working image. This is the one you are to revert to.
+
+- Deploy the known last good working image.
 
-- Revert Production deployments to the last good working image.
   ```
   hokusai production deploy <tag>
   ```
 
-  That updates Kubernetes `Deployments` only. If other Kubernetes resources (e.g. `Ingress`) have to be reverted also, add `update-config` flag:
+  That updates Kubernetes `Deployment` resources only. If other resources (e.g. `Ingress`) should be reverted as well, add `update-config` flag:
+
   ```
   hokusai production deploy <tag> --update-config
   ```
 
-Please note that additional manual steps (e.g. assets compilation, updating configuration) may be required to achieve a complete fix/roll-back, depending on the Project and your situation.
+Please note that `hokusai production deploy...` may not be a comprehensive rollback. Depending on your situation, additional steps (e.g. assets compilation, updating configuration) may be required to achieve a complete fix/roll-back.
```

Comment:
this is now the authoritative source for rollbacks, [hokusai doc's version](https://github.com/artsy/hokusai/blob/main/docs/Command_Reference.md#how-to-rollback) will be removed.